<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1485</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AkademeMate</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="TStart_button">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>550</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Timer Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TStop_button">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>580</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Timer Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TLap_button">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>610</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Timer Lap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TReset_button">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>640</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Timer Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTimer">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>670</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="LapsTable">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>550</y>
      <width>111</width>
      <height>192</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
    <column>
     <property name="text">
      <string>Lap</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Time</string>
     </property>
    </column>
   </widget>
   <widget class="QProgressBar" name="progressBar_All">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>1031</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(205, 255, 247)</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QTableView" name="MainTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>1041</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddModuleButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>720</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Add Module</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EditModuleButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>720</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Edit Module</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteModuleButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>720</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Delete Module</string>
    </property>
   </widget>
   <widget class="QTableView" name="DetailTable">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>40</y>
      <width>421</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AkademeMate</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1485</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
